engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
I was willing to accept the OR SQL for now but when I saw that it was leading to polymorphic parameters I got scared :black_joker: